<paste log output from Kubernetes-mediated prediction, here>


(.devops) smithcf@DESKTOP-DACPO1G:/mnt/c/Users/Connor/Documents/Udacity Nanodegree/Section5/DevOps_Microservices/project-ml-microservice-kubernetes$ ./run_kubernetes.sh
Error from server (AlreadyExists): pods "pybuild1" already exists
NAME       READY   STATUS              RESTARTS   AGE
pybuild1   0/1     ContainerCreating   0          18s
error: unable to forward port because pod is not running. Current status=Pending



(.devops) smithcf@DESKTOP-DACPO1G:/mnt/c/Users/Connor/Documents/Udacity Nanodegree/Section5/DevOps_Microservices/project-ml-microservice-kubernetes$ ./run_kubernetes.sh
Error from server (AlreadyExists): pods "pybuild1" already exists
NAME       READY   STATUS    RESTARTS   AGE
pybuild1   1/1     Running   0          35s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000
Handling connection for 8000

smithcf@DESKTOP-DACPO1G:/mnt/c/Users/Connor/Documents/Udacity Nanodegree/Section5/DevOps_Microservices/project-ml-microservice-kubernetes$ ./make_prediction.sh
Port: 8000
{
  "prediction": [
    20.35373177134412
  ]
}
smithcf@DESKTOP-DACPO1G:/mnt/c/Users/Connor/Documents/Udacity Nanodegree/Section5/DevOps_Microservices/project-ml-microservice-kubernetes$ 
